flask/tests/test_misc.py

```diff
@@ -27,6 +27,16 @@ def test_get_enums(test_database, client, login_as):
         assert enumType is not None
 
 
+# GET /api/metadatasettypes
+def test_get_metadatasettypes(test_database, client, login_as):
+    login_as("admin")
+    response = client.get("/api/metadatasettypes")
+    assert response.status_code == 200
+    assert len(response.get_json()) == 4
+    for metadataset_type, _ in response.get_json().items():
+        assert metadataset_type is not None
```

Comment:
JSON keys cannot be null so this check doesn't do anything. It would be better to assert that the values aren't null and are arrays.